### DISCUSSION TITLE
Вывод записей по тегу в нужном месте

### DISCUSSION THREAD

**Discussion by W1SE95** [maxsite/cms#477] [Q&A] [answered]
Есть такой код:
`$pages2 = mso_get_pages([
    'custom_type' == 'tag',
	'meta_key' => $this_page_tag,
	'type' => false,
	'content' => false,
    ], $pagination2);`

Он работает, но показывается не более 1 записи по тегу, что нужно сделать, чтобы показывать весь список и чтобы работал limit =>?

При указании 'custom_type' **=>** 'tag', - вообще ничего не выводится

P.S. = $this_page_tag - это $this_page_tag = implode(",", $p->val('page_tags'));

**Comment by maxsite** [comment]
Если вы хотите получить записи по нескольким тэгам, то лучше составить отдельный запрос. Потому что mso_get_pages() заточена под входящий slug. А вы хотите передать несколько. 

Вначале получите все метки записи mso_get_tags_page(), потом сделайте запрос к  mso_meta, где укажите условие OR и получите список id записей.

**Reply by W1SE95** [reply]
Не совсем вы верно меня поняли. У каждой записи не более одного тега, так вот именно по 1 тегу и нужно показывать другие записи с этим же тегом.

Пример:
1-я Запись (тег: null1) - родитель
2-я Запись (тег тот же) - child
3-я Запись (тег тот же) - child
4-я Запись (тег тот же) - child
...

**Comment by maxsite** [comment, marked as answer]
Ну тогда используйте 'custom_type' => 'tag' чтобы получить записи через _mso_sql_build_tag() Параметр slug — это и есть метка.

**Reply by W1SE95** [reply]
Все так элементарно! Заработало :) Низкий поклон вам.

**Comment by maxsite** [comment]
Вот здесь: `'meta_key' => $this_page_tag,` не нужно. Просто укажите `'slug' => 'метка' `
